docs/api.md

```diff
@@ -462,6 +463,11 @@ Shortcut for [page.mainFrame().focus(selector)](#framefocusselector).
 #### page.frames()
 - returns: <[Array]<[Frame]>> An array of all frames attached to the page.
 
+#### page.getContent()
```

Comment:
@aslushnikov thank you for the review! I've just updated it to be `.content()` instead of `.getContent()`.